README.md

```diff
@@ -187,6 +119,156 @@ module "spacelift" {
 }
 ```
 
+<!-- BEGIN_TF_DOCS -->
+## Requirements
```

Comment:
I'd say, let's just link the tofu registry, as it contains all this stuff, and automatically kept up to date:

https://github.com/spacelift-io/terraform-google-spacelift-selfhosted/pull/30/files